src/main/java/Duke/Tasks/Task.java

```diff
@@ -0,0 +1,80 @@
+
+package Duke.Tasks;
+/**
+ * Contains the ToDo class
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * The contructor for Task class
+     *
+     * @param description
+     *
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * The method for getSymbol
+     *
+     */
+    public String getSymbol() {
+        return (isDone ? "[X]" : "[ ]"); // mark done task with X
+    }
+
+    /**
+     * The getter method for getting description
+     *
+     */
```

Comment:
Comments could perhaps follow JavaDocs format more closely. (Include @params and @return